pppd/lcp.c

```diff
@@ -67,12 +72,18 @@
 
 static void lcp_delayed_up(void *);
 
+#define LCP_RTT_MAGIC 0x19450425
```

Comment:
I haven't seen anywhere where you have said "round trip time" rather than "RTT". If the reader doesn't happen to be familiar with that acronym, this code could be somewhat obscure. I would like to see a comment here (or some other suitable place) explaining that "These definitions relate to the measurement and logging of round trip time (RTT) of LCP echo-requests" (or something similar). That way the acronym is expanded in at least one place.